sqls/uat/getUatWithCounty.sql

```diff
@@ -0,0 +1,27 @@
+IF EXISTS (
+    SELECT *
+    FROM sysobjects
+    WHERE type = 'P' AND name = 'getUatWithCounty'
+)
+BEGIN
+    DROP PROCEDURE getUatWithCounty
+END
+GO
+
+CREATE PROCEDURE [dbo].getUatWithCounty(@sirutaId INT) AS
```

Comment:
wouldn't be better to just get a single recordset that contains both Uat and County info? I think there are 2 recordset with only one record each. Or am I completely wrong?

Reply:
I found it simpler to do the mapping in JS this way, your solution works as well, but I see the performance gains too small in order to bother.

Reply:
Well, I guess is not as much about the performance gain or loss in this case as it is about the wasted computing cycles on the SQL server. We are running 2 selects that can be easily concatenated into a single one just to combine their results on the backend. Please forgive me but I can't see any reason for not doing just this

```
    SELECT U.sirutaId, U.type, U.name, C.id, C.name
    FROM County C
    INNER JOIN dbo.Uat U on C.id = U.countyId
    WHERE U.sirutaId = @sirutaId
```

but maybe I'm missing something

Reply:
Wasted computing cycles isn't about slower performance? I see your point, don't get me wrong and I already underlined my reasons for writing 2 queries instead of one: I've sacrificed performance for easier-to-write BE code. Anyway, please check now.